_layouts/default.html

```diff
@@ -14,13 +14,21 @@
 
     {% include footer.html %}
     <script type="text/javascript" src="{{ site.baseurl }}/assets/js/bundle.js" charset="utf-8"></script>
-    <!-- Including the bootstrap css via CDN -->
-    <link 
-      rel="stylesheet" 
-      href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" 
-      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" 
-      crossorigin="anonymous"
-    >
+    
+    <!-- Include JQuery and Bootstrap JS -->
+    <script
```

Comment:
Why would these not be done via webpack?

Reply:
Because I didn't realize this site uses React-Bootstrap which doesn't depend on those scripts. :sweat_smile: 

I am not really paying attention to this repo since I don't know what it's about + focusing on the IMU stuff.